### DISCUSSION TITLE
Issue with installing controlnet on A1111

### DISCUSSION THREAD

**Discussion by nsikaria** [Mikubill/sd-webui-controlnet#2488] [Q&A]
I am new to A1111 or any other AI stable diffusion..
Recently installed A1111 with OpenVINO script for intel integrated graphics, now I am trying to install ControlNet extension, but it is showing an error as 
In Stable Diffusion page:  PermissionError: [WinError 5] Access is denied: 'G:\\A1111\\stable-diffusion-webui\\tmp\\sd-webui-controlnet' -> 'G:\\A1111\\stable-diffusion-webui\\extensions\\sd-webui-controlnet'

In Web-ui CMD command -
*** Error completing request
*** Arguments: ('', 'https://github.com/Mikubill/sd-webui-controlnet.git', '') {}
    Traceback (most recent call last):
      File "G:\A1111\stable-diffusion-webui\modules\call_queue.py", line 57, in f
        res = list(func(*args, **kwargs))
      File "G:\A1111\stable-diffusion-webui\modules\ui_extensions.py", line 638, in <lambda>
        fn=modules.ui.wrap_gradio_call(lambda *args: [gr.update(), *install_extension_from_url(*args)], extra_outputs=[gr.update(), gr.update()]),
      File "G:\A1111\stable-diffusion-webui\modules\ui_extensions.py", line 385, in install_extension_from_url
        raise err
      File "G:\A1111\stable-diffusion-webui\modules\ui_extensions.py", line 377, in install_extension_from_url
        os.rename(tmpdir, target_dir)
    PermissionError: [WinError 5] Access is denied: 'G:\\A1111\\stable-diffusion-webui\\tmp\\sd-webui-controlnet' -> 'G:\\A1111\\stable-diffusion-webui\\extensions\\sd-webui-controlnet'

I have tried reinstalling with cmd administrator access also, closing file explorer but then also it is not installing.

Any other extension downloads just fine..

**Comment by huchenlei** [comment]
Issue filed in A1111 https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/14682
